MetaStation bar remake

ClownMoff opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

New, original look for MetaStation service and its maintenance

## Why It's Good For The Game

The new Meta bar we merged on https://github.com/BeeStation/BeeStation-Hornet/pull/13291 was heavily flawed, it was a /TG/ port that needed some work and got merged without a proper, extensive review. The map is missing firelocks and it's just not up to quality mapping standards.  Instead of having yet another /TG/ port, let's use our own map.

You may find in-game screenshots below.

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>

<img width="416" height="742" alt="image" src="https://github.com/user-attachments/assets/0dabb94c-3b05-428d-b428-e86199c2fc2a" />
<img width="783" height="875" alt="image" src="https://github.com/user-attachments/assets/db3d79c3-c34f-47b6-868a-e2e7b92433c7" />
<img width="758" height="671" alt="image" src="https://github.com/user-attachments/assets/d2975b15-1b25-4dc2-8baf-faa925873da4" />
<img width="859" height="569" alt="image" src="https://github.com/user-attachments/assets/717e65b1-8086-4a7f-9700-5c2b536392ef" />
<img width="567" height="645" alt="image" src="https://github.com/user-attachments/assets/91cce94f-2120-4c9e-b22a-cce8b2ca817e" />

</details>

## Changelog
:cl: Varo10, spockye
tweak: Remade MetaStation service area
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
lets see if linters go through

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
linter fix

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
more lints and small stuff

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
lintsisters

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

mrmanlikesbt reviewed (dismissed):
looks good!

a few things I noticed:
- no thunderdome viewing consoles /obj/machinery/computer/security/telescreen/entertainment
- the only botanist spawner is the one in the backroom
- no security cameras
- no holopads
- seemingly no airlock/windoor for the kitchen freezer + the disposal unit is in the way
- the bar/engineering tile decals in the hallway above the bar don't line up with the doors and there's a stray yellow decal
- no gambling spawners on the tables or cardpacks or whatever
- detective spawn at the bar? quirky, but lets keep it in the detective's office
- improperly placed air alarm helper in the freezer
- use the new kitchen air alarm helper /obj/effect/mapping_helpers/airalarm/tlv_kitchen
- test walking around the stage in-game. ledges/platforms/whatever they're called weren't properly implemented or tested and they can act weird- totally blocking movement. you can see this on your corgstation bar
- stray decals on the southern hydroponics wall
- no chef clothing vendor
- bring back the kitchen shutters

non blocking requests:
- the firelocks above the bar now look odd sitting up against the windows, it would probably look better if they were pushed back to the wall
- the stage looks a little bland, maybe you could add like... idk
- maybe add some posters by the theatre stage?
- why would you give the bartender a door to the freezer if you're just gonna bolt it?
- adding some status displays would be nice, /obj/machinery/status_display/evac and /obj/machinery/status_display/ai

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
pipe work ALREADY PLEASE

spockye reviewed (changes requested):
<details>
  <summary>REVIEW CONTAINED IN HERE</summary>

 
<img width="763" height="857" alt="image" src="https://github.com/user-attachments/assets/d70d7834-efba-4d8b-929d-f3fe65527bf9" />

Piping errors

<img width="685" height="652" alt="image" src="https://github.com/user-attachments/assets/9858d945-64ee-46cc-96bc-b2dee4057dd9" />

disposals errors (remember to go ingame and check it!)

<img width="913" height="894" alt="image" src="https://github.com/user-attachments/assets/72d3e3ab-67f9-493b-8659-2801153c5d2f" />

wiring errors

<img width="668" height="559" alt="image" src="https://github.com/user-attachments/assets/c7d13fb8-0b50-4a39-846d-d4c7b2d94798" />

Fill yo corners foo

----------------------------------

This is nice, good work! i'd like to see this tm'd after fixes are through. keep it up!

</details>

ClownMoff committed:
Marty Fixes

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

PowerfulBacon reviewed (approved):
Looks good in game

mrmanlikesbt reviewed (changes requested):
These pipes should be hidden.

<img width="355" height="476" alt="image" src="https://github.com/user-attachments/assets/dbeaed7e-ff55-49b1-9339-20bbd23cb384" />

Incorrect air alarm helper. Should be `tlv_cold_room` instead of `tlv_kitchen`

<img width="349" height="504" alt="image" src="https://github.com/user-attachments/assets/4a91a779-1ebf-41ba-b5c0-45d77de1986d" />

No air alarm helper here. Should be `tlv_kitchen`

<img width="258" height="220" alt="image" src="https://github.com/user-attachments/assets/0647ed15-752d-4a23-a738-48ed945b915d" />

camera

<img width="371" height="378" alt="image" src="https://github.com/user-attachments/assets/c2921966-9054-4493-beb1-883a21b71584" />

This ledge looks weird in-game

<img width="188" height="266" alt="image" src="https://github.com/user-attachments/assets/60686b2a-6247-42b1-83eb-b6f2880120a8" />

the access helper here is wrong. it should be `/obj/effect/mapping_helpers/airlock/access/all/service/bar`, not `/obj/effect/mapping_helpers/airlock/access/any/service/maintenance`. the one you have it set to checks for either maintenance access or general service access

<img width="259" height="497" alt="image" src="https://github.com/user-attachments/assets/0cbc7954-6e57-4c71-94dd-63b67492ffdf" />

goofy window

<img width="259" height="322" alt="image" src="https://github.com/user-attachments/assets/ed6eaca3-91ab-4976-9e4c-2c802ed788ca" />

avoid placing two wall mounted objects on the same wall. it looks bad when observing

<img width="183" height="203" alt="image" src="https://github.com/user-attachments/assets/96c7e1a9-ccc0-44ae-a7ee-2012ecbe3887" />

re-implement the relevant fixes in #13324 ping me on discord if you need specifics

the vast majority of the airlocks you placed have their access varedited. get rid of the these.

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Create MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
sink

spockye reviewed (changes requested):
<img width="670" height="635" alt="Image" src="https://github.com/user-attachments/assets/bc09b832-ee44-41b2-a115-71ae7503f3ea" />

piping errors

<img width="742" height="680" alt="Image" src="https://github.com/user-attachments/assets/b504ab10-1ebf-45bf-a398-c94479639c25" />

Power errors
(and wild floortile)

<img width="376" height="248" alt="Image" src="https://github.com/user-attachments/assets/8dc82798-d85e-4ba3-ac67-c0f9be7981f5" />

area errors

<img width="353" height="302" alt="Image" src="https://github.com/user-attachments/assets/34b327e2-c350-4d7c-9a50-c1e6128f909a" />

wild disposal

<img width="606" height="817" alt="Image" src="https://github.com/user-attachments/assets/a33a9cb1-6ab7-4c1a-bcd2-e59c69dacd2c" />

wild decals

-------------------

overall review
--

so, firstly this is very nice! couple issues here and there but overall i like this change. much better to have our own original works. 

haven't taken a look ingame yet, but i'd like to see this TM'd after you've fixed the issues pointed out

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Update MetaStation.dmm

mrmanlikesbt reviewed (changes requested):
two cameras right next to each other and wallmounts obscuring each other

<img width="420" height="418" alt="image" src="https://github.com/user-attachments/assets/2ac82c3b-0b1d-438e-9f33-dcbee81b80ef" />

something is suppose to be here methinks

<img width="252" height="143" alt="image" src="https://github.com/user-attachments/assets/75b228e0-9407-43cb-a5fb-08f7e02ff552" />

could be more light switches in hydroponics

<img width="819" height="506" alt="image" src="https://github.com/user-attachments/assets/fdad5447-ef69-4680-9314-17701d5b8e87" />

fire extinguisher on glass, evil evil evil evil

<img width="158" height="193" alt="image" src="https://github.com/user-attachments/assets/c007c5ae-bf58-46ab-b9d6-2ac584c8a813" />

conflicting wallmounts

<img width="170" height="141" alt="image" src="https://github.com/user-attachments/assets/b193a940-6389-4945-a658-c98b54197b0b" />

freezer scrubs on roundstart, can the condimaster even be reached?

<img width="388" height="517" alt="image" src="https://github.com/user-attachments/assets/45ac466b-bc50-4bb6-a5f9-29e2737b9adb" />

why are these mismatched tables? it looks off in-game

<img width="102" height="238" alt="image" src="https://github.com/user-attachments/assets/92205b82-bedc-4899-8bb5-dd92d147f14a" />

empty table

<img width="123" height="135" alt="image" src="https://github.com/user-attachments/assets/937f829a-6c1b-42bd-bd1f-b9341407d563" />

table in corner

<img width="197" height="178" alt="image" src="https://github.com/user-attachments/assets/8188ab9b-203a-47d1-aca0-6784e1d75ab5" />

this looks... weird

<img width="173" height="187" alt="image" src="https://github.com/user-attachments/assets/8cd8df30-34d9-41ad-93bf-c7827216b83b" />

weird pixel offset disposal chute?

<img width="140" height="126" alt="image" src="https://github.com/user-attachments/assets/af0f557c-b697-48be-843d-2a3b9da02a56" />

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Update MetaStation.dmm

mrmanlikesbt committed:
Merge remote-tracking branch 'upstream/master' into pr/13346

mrmanlikesbt reviewed (approved):
I see no more issues. You're in the hands of @spockye now

ClownMoff committed:
Merge remote-tracking branch 'upstream/master' into meta-services-fixes

ClownMoff committed:
Merge branch 'meta-services-fixes' of https://github.com/ClownMoff/BeeStation-Hornet into meta-services-fixes

spockye committed:
Merge remote-tracking branch 'upstream/master' into pr/13346

ClownMoff committed:
Update MetaStation.dmm

ClownMoff committed:
Merge branch 'meta-services-fixes' of https://github.com/ClownMoff/BeeStation-Hornet into meta-services-fixes

spockye committed:
Merge remote-tracking branch 'upstream/master' into pr/13346

spockye committed:
let's get this merged

Fixes:
Couple missing pips
pipes not connected to anything
Wires going nowhere
missing wires
missplaced decals
floorturfs
--------
everything should be fine now. Good work, again!

spockye committed:
camera from mr'mans review

spockye reviewed (approved):
fixed a couple remaining issues as we spoke about. 
This is a definite improvement
good work!
